front-end/src/App.js

```diff
@@ -26,6 +28,8 @@ const App = () => {
 						<Route path="/course-details" element={<CourseDetails />} />
 						<Route path="/register/:courseName" element={<RegisterCourse />} />
 						<Route path="/user" element={<UserInformationScreen />} />
+						<Route path="/Login" element={<Login />} />
```

Comment:
I don't think there is a need to include the Login or Landing Page components, they are not yet being used anywhere and so we don't need a URL to access them.

Reply:
yes, thats true.